README.md

```diff
@@ -726,15 +727,22 @@ Modify `ModifyChunkResponsePlugin` to your taste. Example, instead of sending ha
 
 This plugin uses `Cloudflare` hosted `DNS-over-HTTPS` [API](https://developers.cloudflare.com/1.1.1.1/encrypted-dns/dns-over-https/make-api-requests/dns-json) (json).
 
-Start `proxy.py` as:
+`DoH` mandates a HTTP2 compliant client. Unfortunately `proxy.py`
+doesn't provide that yet, so we use a dependency. Install it:
+
+```bash
```

Comment:
Since this has a prompt, use
```suggestion
```console
```

Reply:
I see, then ideally there are several changes to be done in this file.  Quite a few example with prompts are marked as bash currently.